src/go/rpk/pkg/cli/cmd/wasm/common.go

```diff
@@ -105,3 +113,21 @@ func CreateRemoveMsg(name string) sarama.ProducerMessage {
 		Headers: headers,
 	}
 }
+
+func AddTypeFlag(command *cobra.Command, coprocType *string) {
+	command.Flags().StringVar(
+		coprocType,
+		"type",
+		"async",
+		"Coproc type engine. Can be: async(default), data-policy",
```

Comment:
Can this change to `Coproc type engine (async, data-policy)`?

The default is displayed by cobra automatically.

Reply:
Done, thanks for default param!